plugins/modules/identity/ipa/ipa_otpconfig.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Ansible Project
+# Heavily influenced from Fran Fitzpatrick <francis.x.fitzpatrick@gmail.com> ipa_config module
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ipa_otpconfig
+author: justchris1 (@justchris1)
+short_description: Manage FreeIPA OTP Configuration Settings
+version_added: 2.5.0
+description:
+- Modify global configuration settings of a FreeIPA Server with respect to OTP (One Time Passwords).
+options:
+  ipatokentotpauthwindow:
+    description: TOTP authentication window in seconds.
+    aliases: ["totpauthwindow"]
+    type: int
+  ipatokentotpsyncwindow:
+    description: TOTP synchronization window. (Seconds)
```

Comment:
This (and the fourth option) need similar adjustments to the other two.

Reply:
Resolved in 59f47ff